go-backend/db/organization.go

```diff
@@ -131,41 +146,47 @@ func (s *pgStore) CreateOrganization(ctx context.Context, org Organization) (cre
 		org.Hi5Limit,
 		org.Hi5QuotaRenewalFrequency,
 		org.Timezone,
-	).Scan(&lastInsertId)
+		org.CreatedAt,
+	).Scan(&lastInsertID)
 	if err != nil {
 		logger.WithField("err", err.Error()).Error("Error creating organization")
 		return
 	}
 
-	err = s.db.Get(&createdOrganization, getOrganizationQuery, lastInsertId)
-
+	err = s.db.Get(&createdOrganization, getOrganizationQuery, lastInsertID)
+	if err != nil {
+		if err == sql.ErrNoRows {
+			// TODO: Log that we can't find the organization even though it's just been created
```

Comment:
what about this TODO?

Reply:
3c14d8138147b587a472dd81f242face53718c55 contains a fix